tests/e2e/main_test.go

```diff
@@ -106,6 +107,8 @@ func (s *BabylonSDKTestSuite) Test1ContractDeployment() {
 	msgUpdateParams := &bbntypes.MsgUpdateParams{
 		Authority: s.ConsumerApp.BabylonKeeper.GetAuthority(),
 		Params: bbntypes.Params{
+			FinalityInflationRate:      sdkmath.LegacyNewDecWithPrec(7, 2),
+			BlocksPerYear:              60 * 60 * 25 * 365 / 5, // 5 seconds per block
```

Comment:
This can be obtained from `x/mint`'s `MintKeeper` so the Babylon SDK module does not need to have it

Reply:
OK, will refactor.